extensions/vercte/dictionaries.js

```diff
@@ -0,0 +1,142 @@
+window.dictionaries = new Map();
```

Comment:
in addition to all the other warnings, extensions cannot store things on `window`; use the IIFE from https://docs.turbowarp.org/development/extensions/unsandboxed then just define `var dictionaries = `